models/cnn/AlexNet/test_AlexNet.py

```diff
@@ -29,7 +30,148 @@ def test_overfit(model):
 
 
 if __name__ == "__main__":
+    # num_classes = 10
+    #case 1 
     model = AlexNet(num_classes=10)
     model = model.build(input_shape=(128, 128, 3))
     test_forward_pass(model)
     test_overfit(model)
+    #case 2 
```

Comment:
For running multiple test cases, use @pytest.mark.parametrize as I did for test_SegNet.py.